projects/plugins/jetpack/extensions/store/media-source/README.md

```diff
@@ -0,0 +1,53 @@
+Media Source State Tree
+=======================
+
+This subtree takes over registering media sources globally in the editor app context. Initially created with the purpose to put all components together of the anchor.fm integration provides a communication channel allowing controlling, for instance, the audio state from `play` to `stop`, and vice versa.
```

Comment:
> Initially created with the purpose to put all components together of the anchor.fm integration provides a 

Maybe we could rephrase this to something like:

Initially created for the anchor.fm integration, it provides a communication channel for controlling, for instance, ...

Reply:
I've changed the description, but it's something we'd like to keep updated with the API, so probably going to change soon.